kraken/kubernetes/client.py

```diff
@@ -1,4 +1,5 @@
 from kubernetes import client, config
+from kubernetes.stream import stream
```

Comment:
Add kubernetes and oauth2client to requirements.txt.  Interesting no one caught that before.

Reply:
It looks like they are installed from the powerfulseal installation. Should I also add to our requirements? They seem to be using a specific version of kubernetes (kubernetes==12.0.0a1)

Reply:
On a RHEL 7.5 jump host I created a new virtualenv and did a pip install -r and they did not get installed.   Strange.  I had to install them explicitly.   I think it is OK if we add them to be save kubernetes>=12.0.0a1 and oauth2client>=whatever you have.   Mine is 4.1.3.

Reply:
Added!